start.sh

```
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Wait for database to be ready and create tables
+echo "🔄 Iniciando aplicación..."
+
+# Create database tables
+echo "📊 Creando tablas de base de datos..."
+python -m app.db.config.create_tables
+
+if [ $? -ne 0 ]; then
```

Shellcheck (0.10.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)